src/test/restart.test.ts

```diff
@@ -0,0 +1,36 @@
+import * as vscode from 'vscode'
+
+import {
+  getDocUri,
+  testDiagnostics,
+  toRange,
+  waitForDiagnosticChange
+} from './utils'
+
+suite('Should get diagnostics', () => {
+  const docUri = getDocUri('test.ts')
+  test('Get ast-grep issues', async () => {
+    await vscode.commands.executeCommand('ast-grep.restartLanguageServer')
```

Comment:
curious about what's the difference between simple diagnostics test? restart seems to only check the lsp still works like before. but the new config change is not reflected.

Reply:
Yes...there is no difference. After adding fileModificationWatcher, this behavior becomes difficult to test.

The scenario where I often use this kind of command is that TypeScript does not monitor changes in the node_modules, so I will restart the ts language server after install at all time.

Reply:
I hate fileModificationWatcher and flaky test. Removing it